src/main/java/com/sprints/model/Sprint.java

```diff
@@ -1,5 +1,103 @@
 package com.sprints.model;
 
+import java.util.Date;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Document(collection="sprints")
 public class Sprint {
+	@Id
+	String id;
+	public String getId() {
```

Comment:
Remove **all** getters, setters & constructors and use Lombok annotations.

Reply:
Done